Deliverable2/Program.cs

```diff
@@ -85,75 +82,70 @@ public static void Main(string[] args)
                         Case 2 = User Declined A Drink 
                         */
                         Console.ForegroundColor = ConsoleColor.Green;
-                        if (StartOverCase == 0)
+                        if (startOverCase == 0)
                             
                         {
                             Console.WriteLine("Alright Guest " + i + ", would you like water or a softdrink? \n" +
                                             "(You can type w or s for faster service)");
-                            AddPadding(Padding);
+                            AddPadding(padding);
                         }
-                        else if (StartOverCase == 1)
+                        else if (startOverCase == 1)
                         {
                             Console.WriteLine("Ok lets try this again Guest " + i + ",\n" +
                                             "would you like water or a softdrink? (You can type w or s for faster service)");
-                            StartOverCase = 0;
-                            AddPadding(Padding);
+                            startOverCase = 0;
+                            AddPadding(padding);
                         }
-                        else if (StartOverCase == 2)
+                        else if (startOverCase == 2)
                         {
                             Console.WriteLine("Ok  Guest " + i + " orders nothing... Moving on.");
-                            StartOverCase = 0;
-                            AddPadding(Padding);
-                            GuestSelection = "none";
+                            startOverCase = 0;
+                            AddPadding(padding);
+                            guestSelection = "none";
                         }
                         Console.ForegroundColor = ConsoleColor.White;
 
-                        if (GuestSelection == "none")
```

Comment:
this section wasn't doing anything important because we already know that GuestSelection == "non"